src/documents/tutorial-developerIntroduction/DeveloperIntroductionToInfusionFramework-Components.md

```diff
@@ -0,0 +1,43 @@
+---
+title: Developer Introduction to the Infusion Framework - Components
+layout: default
+category: Tutorials
+---
+
+Programs using Infusion structure their data, functionality and interaction as various components defined using a common syntax.
+
+Components are [JavaScript objects that have certain characteristics](/infusion/development/UnderstandingInfusionComponents.md). Components can be freely combined together to form new components using the [grades system](/infusion/development/ComponentGrades.md), or organized into relationships with one another via [subcomponents](/infusion/development/SubcomponentDeclaration.md).
```

Comment:
All of these links are broken because they are deployed to a subdirectory.  Long-term, I can imagine us wanting to have draft docs, versioned docs, both of which would likely need to live in some kind of subpath rather than the root.  We also want it to be possible to view work-in-progress on GitHub pages and other static sites.  I'd suggest searching for '(/infusion/development' and replacing with '(../' with case-by-case review.